docs/overview/explore-with-script-lab.md

```diff
@@ -56,10 +56,20 @@ Script Lab is supported for Excel, Word, and PowerPoint on the following clients
 - Office 2016 or later on Mac
 - Office on the web
 
+Script Lab for Outlook is available on the following clients.
+
+- Outlook 2013 or later on Windows
+- Office 2016 or later on Mac
+- Office on the web when using Chrome, Microsoft Edge, or Safari browsers.
+
+For complete details on Script Lab for Outlook, see the [Script Lab now supports Outlook blog post](https://developer.microsoft.com/en-us/office/blogs/script-lab-now-supports-outlook/) available on the Office 365: Developer Blog.
+
 ## Next steps
 
 To use Script Lab in Excel, Word, or PowerPoint, install the [Script Lab add-in](https://appsource.microsoft.com/product/office/WA104380862) from AppSource. 
 
+To use Script Lab for Outlook, install the [Script Lab for Outlook add-in](https://appsource.microsoft.com/en-US/product/office/wa200001603?tab=overview) from AppSource.
+
 You're welcome to expand the sample library in Script Lab by contributing new snippets to the [office-js-snippets](https://github.com/OfficeDev/office-js-snippets#office-js-snippets) GitHub repository.
 
 When you're ready to create your first Office Add-in, try out the quick start for [Excel](../quickstarts/excel-quickstart-jquery.md), [Outlook](../quickstarts/outlook-quickstart.md), [Word](../quickstarts/word-quickstart.md), [OneNote](../quickstarts/onenote-quickstart.md), [PowerPoint](../quickstarts/powerpoint-quickstart.md), or [Project](../quickstarts/project-quickstart.md).
```

Comment:
In the See also section, there's a link to "Get Script Lab". We'll need to add an entry to "Get Script Lab for Outlook".